dev-packages/e2e-tests/test-applications/nextjs-14/tests/request-instrumentation.test.ts

```diff
@@ -13,7 +13,7 @@ test('Should send a transaction with a fetch span', async ({ page }) => {
       data: expect.objectContaining({
         'http.method': 'GET',
         'sentry.op': 'http.client',
-        'sentry.origin': 'auto.http.node.undici',
+        'next.span_type': 'AppRender.fetch', // This span is created by Next.js own fetch instrumentation
```

Comment:
l: We could think about adding `sentry.origin: 'auto.http.nextjs.fetch'` or something like this if we detect this.

Reply:
I have started a broader discussion around this topic last week because we need a generic decision in all SDKs for this. The important question being: "What sentry.origin do we give spans that are not generated through Sentry SDK API?" Right now it is manual - which is obv wrong. I'd argue we don't set anything, because sentry.origin doesn't make sense anymore.